lib/host-sp-messages/src/lib.rs

```diff
@@ -266,6 +270,11 @@ pub enum InventoryData {
         mfr_serial: [u8; 20],
         /// MFR_FIRMWARE_DATA, PMBus operation 0xFD
         mfr_firmware_data: [u8; 20],
+
+        temp_sensor: SensorId,
```

Comment:
As we add more types of sensors to these devices of the same class, for example, there are two temp sensors, how do we end up retaining the mapping of which sensor is which in these commands? Or perhaps on the addition of synthetic sensors. I think this probably ties into the other question I have more generally about how is this mapping constructed and then how does it get maintained.

Reply:
We talked about this in person, but quick summary for the record – this is similar to [the previous discussion](https://github.com/oxidecomputer/hubris/pull/1514#discussion_r1315995020), where right now it's manual / dependent on ordering in the TOML file.  Once we create synthetic sensors, we'll want to revisit this, but it shouldn't block integration of this PR.